src/server/main_service.cc

```diff
@@ -1133,6 +1134,13 @@ void Service::DispatchCommand(CmdArgList args, facade::ConnectionContext* cntx)
     if (auto& exec_info = dfly_cntx->conn_state.exec_info; exec_info.IsCollecting())
       exec_info.state = ConnectionState::ExecInfo::EXEC_ERROR;
 
+    // We need to skip this because ACK's should not be replied to
+    // Bonus points because this allows to continue replication with ACL users who got
+    // their access revoked and reinstated
+    if (cid->name() == "REPLCONF" && absl::EqualsIgnoreCase(ArgS(args_no_cmd, 0), "ACK")) {
+      LOG(INFO) << "Tried to reply to REPLCONF";
```

Comment:
use LOG(ERROR) and print err 
you can also add DCHECK so this flow will crash if we run in debug mode

Reply:
I don't want this to be a failure(DCHECK), and it's not always an error. This *will* trigger if we revoke the ACL's because internally we send REPLCONF ACK which will `fail`. When this happens, `VerifyCommandState` will return an error, and the path will try to reply with a `SendError`. It's better if we keep it as `LOG(INFO)` because:

1. It's not always an error
2. DCHECK will make this path fail which is wrong, because this path will get exercised each time we revoke REPLCONF from an ACL user.

Reply:
We just need to skip the reply if it's a `REPLCONF` and that's pretty much it. I want the LOG in case we break it somehow (or if there is any path that tries to reply that we are not aware of)